src/cdk/a11y/key-manager/list-key-manager.ts

```diff
@@ -72,6 +78,16 @@ export class ListKeyManager<T extends ListKeyManagerOption> {
   /** Stream that emits whenever the active item of the list manager changes. */
   change = new Subject<number>();
 
+  /**
+   * Predicate function that determines items that should be skipped by the list key manager.
+   * By default, disabled items are skipped by the key manager.
```

Comment:
This sounds like it's describing the `predicate` argument, instead of what the method does. Consider changing it to something like "Sets the predicate function that the determines..."?